whats_fresh/whats_fresh_api/templates/list.html

```diff
@@ -53,3 +62,7 @@ <h2 class='light'>{{ title }}</h2>
 </div>
     </div>
 {% endblock content %}
+
+<script>
+$('#search_list').attr("value", "kokab");
```

Comment:
Oops, this was when I was trying to retain the value of searches!
Will remove it :p